### ISSUE TITLE
try using the argument 'weight' of loss function

### ISSUE THREAD

**Issue by siruku6** [siruku6/fx_notebooks#2]
https://stackoverflow.com/questions/61414065/pytorch-weight-in-cross-entropy-loss

**Comment by siruku6** [comment]
I've done it.
